src/server/ranch/RanchDirector.cpp

```diff
@@ -1779,7 +1779,8 @@ void RanchDirector::HandleUpdatePet(
         character.pets().emplace_back(petUid);
       }
 
-      if (command.actionBitset == protocol::AcCmdCRUpdatePet::Action::Rename)
+      if (command.actionBitset == protocol::AcCmdCRUpdatePet::Action::Rename || 
+          command.actionBitset == protocol::AcCmdCRUpdatePet::Action::RenameToo)
```

Comment:
If changed to bit set enum, the correct way to check is to do:
```suggestion
      if (command.actionBitset & static_cast<uint16_t>(protocol::AcCmdCRUpdatePet::Action::Rename))
```